### PR TITLE
graphical representation of user statistics

### LINKED ISSUE #316: Profile pages - part two

### PR DESCRIPTION
**Pull request by Knorrke**
Represent statistics of user profiles as birdnest. Closes #316

### CODE REVIEW

File: src/assets/source/scripts/main.js [hunk 1]
```diff
@@ -74,6 +75,9 @@ require.config({
         underscore: {
             exports: '_'
         },
+        d3: {
+            exports: 'd3'
```
Comment (inyono) [inline src/assets/source/scripts/main.js]: Is this needed? I think `serlo_math_puzzle` uses `d3`, too, and didn't have this line

**Comment by inyono** [comment]
needs rebase because of sms banners, sorry :/

**Comment by arekkas** [comment]
LGTM

File: src/assets/source/scripts/main.js (hunk 1 above)
Comment (Knorrke) [inline src/assets/source/scripts/main.js]: You're right, seems to be unnecessary. Used an older version (3.1.) before, where this was necessary. I removed it and it still works fine.

**Comment by Knorrke** [comment]
> needs rebase because of sms banners, sorry :/
done

**Pull request merged**